streams/src/main/java/org/apache/kafka/streams/kstream/WindowedKStream.java

```diff
@@ -71,6 +73,45 @@
     KTable<Windowed<K>, Long> count();
 
     /**
+     * Count the number of records in this stream by the grouped key and the defined windows.
+     * Records with {@code null} key or value are ignored.
+     * The specified {@code windows} define either hopping time windows that can be overlapping or tumbling (c.f.
```

Comment:
This is not introduced by this PR: we do not have a parameter `windows` anymore in these functions, so I'm wondering if we should extract this part of the explanation on window specs to the javadoc of the `WindowedKStream` class itself?